examples/sir.jl

```diff
@@ -94,10 +93,10 @@ function model_initiation(;
         reinfection_probability,
         detection_time,
         C,
-        death_rate
+        death_rate,
     )
     space = GraphSpace(complete_graph(C))
-    model = StandardABM(PoorSoul, space; agent_step!, properties, rng)
+    model = StandardABM(PoorSoul, space; agent_step!, model_step!, properties, rng)
```

Comment:
can't you remove this change as well now?

Reply:
sure, I assumed there was one before, that's what happens when you are multitasking xD